tools/generate_project_statistics_csv.php

```diff
@@ -1,29 +1,19 @@
 #!/usr/bin/env php
-<?php
+<?php declare(strict_types=1);
 /**
- * @author Stella Lee
- * Date: 15-08-15
+ * Generates a CSV report of various statistics about this study.
  *
- * Project Statistics - Updating Counts From Each LORIS
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  */
+require_once 'generic_includes.php';
+require_once __DIR__ . '/../php/libraries/OutputWrapper.class.inc';
```

Comment:
This is in the php/libraries directory, it should be autoloaded.

Reply:
@driusan Fixed.